src/test/java/hw8/beans/ExampleData.java

```diff
@@ -0,0 +1,63 @@
+package hw8.beans;
+
+import com.google.gson.annotations.Expose;
+
+import java.util.List;
+
+public class ExampleData {
```

Comment:
This is not an example data, name of the class should be selected with reference to it business purpose.

Reply:
Renamed